glances/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  glances:
+    image: ${GLANCES_IMAGE}
+    restart: unless-stopped
+    pid: host
+    privileged: true
+    network_mode: "host"
```

Comment:
Why is network mode host? This is insecure for traefik.

Reply:
ChatGPT tells me Glances needs to monitor the host's network interfaces. Using host network mode enables it to see and track real-time traffic on the host's network interfaces without being isolated within a container-specific virtual network. And if you are running other services on the host machine that Glances needs to monitor, such as local databases or web servers, host mode allows the container to interact with them directly without needing complex network configurations or port mappings.

I installed without `network_mode: "host"` and Homepage can't see Glances (or more likely, Glances can't see the host so it has nothing to report).

Reply:
ok maybe I need to relent on both of these issues, and just let it do its thing. We need a huge warning on the top of the README that this is essentially a system root level container.

Reply:
```
## Warning

This container uses the docker flags `privileged: true` and
`network_mode: host`. This gives Glances unlimited access to your host
system. You should not install this unless you completely trust this
service.
```